### ISSUE TITLE
Component > Content Switcher > #a11y contrast issue

### ISSUE THREAD

**Issue by arbydev** [carbon-design-system/carbon#340]
http://carbondesignsystem.com/components/content-switcher/code
The hover color does not meet AA contrast requirements with the light blue background and white text.

In Firefox 56.0 on a PC a black dotted outline is appearing within the buttons as I tab through them. moz-focus-inner may help remove that. 

Your choice of light blue border does not have significant contrast. You might consider using a similar treatment to your Buttons with an inner border. You are underlining the text on focus so technically you still have a highly visible focus indicator with that.

**Comment by bsonefeld** [comment]
Hey @arbydev , thanks for your comment!

We are doing a large hover audit on each component as a future task item, so we will keep this in mind. Could you point us to some helpful guidelines on `:hover` color contrast states? Based on the W3C [guidelines](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=141%2C143#visual-audio-contrast-contrast), I wasn't seeing anything specifically on `:hover`. 

For the `:focus` states, what tool are you using to check for "significant contrast"? I'd like to better understand what tests this component is passing or failing.

**Comment by marijohannessen** [comment]
Closing this issue due to lack of response from issue creator. Please feel free to re-open if needed.
